simpledb/recovery.go

```diff
@@ -240,7 +240,9 @@ func (db *DB) replayAndSetupWriteAheadLog() error {
 		err = executeFlush(db, memStoreFlushAction{
 			memStore: swapMemstore(db),
 		})
-
+		if err != nil {
+			log.Printf("Error Flush memstore %v", err)
+		}
```

Comment:
oof, well spotted. Do you think we should return an error here?

Reply:
yes, modified.